rollout.gemspec

```diff
@@ -20,12 +20,13 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3'
 
-  spec.add_dependency 'redis', '~> 4.0'
+  spec.add_dependency 'redis', '~> 3.0'
```

Comment:
Any reason you're downgrading redis?

Reply:
Yes, I tested using on our app and I had conflict with the versions